Scripts/RemediationScripts/Remediate-ConfigureConditionalAccessPolicyForPIM.ps1

```diff
@@ -600,17 +661,16 @@ function Disable-ConditionalAccessPolicyForPIM
         $skippedRoles = @()
         $rolledBackSummary = New-Object -TypeName PSObject
 
-        foreach ($role in $configuredPolicyDetails.RemediatedRoles.Split(','))
+        Write-Host "`nRunning this script will disable all the Conditional Access (CA) policies for the previously remediated role(s). Do you want to continue? " -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
+        $userInput = Read-Host -Prompt "(Y|N)"
+        if($userInput -ne "Y")
         {
-            Write-Host "`nDo you want to disable Conditional Access (CA) policy for [$($role)] role?" -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
-            
-            $userInput = Read-Host -Prompt "(Y|N)"
-            if($userInput -ne "Y")
-            {
-                Write-Host "Conditional Access policy will not be disabled for [$($role)] role. Exiting..." -ForegroundColor $([Constants]::MessageType.Update)
-                continue
-            }
+            Write-Host "Conditional Access policy will not be disabled for any of the roles [User permission denied]. Exiting..." -ForegroundColor $([Constants]::MessageType.Update)
+            return
+        }
 
+        foreach ($role in $configuredPolicyDetails.RemediatedRoles.Split(','))
```

Comment:
If MFA was enabled and we disabled to configure our CA policy, during role back are we setting up old MFA policy again?

Reply:
It was not there earlier, added now.